packages/edge/src/edge-headers.ts

```diff
@@ -68,6 +74,23 @@ function getHeader(request: Request, key: string): string | undefined {
   return request.headers.get(key) ?? undefined;
 }
 
+/**
+ * Converts the 2 digit countryCode into a flag emoji by adding the current character value to the emoji flag unicode starting position.
+ *
+ * @param countryCode The country code returned by: `getHeader(request, COUNTRY_HEADER_NAME)`.
+ * @returns A flag emoji or undefined.
+ */
+function getFlag(countryCode: string | undefined): string | undefined {
+  const regex = new RegExp('^[A-Z]{2}$').test(countryCode!);
+  if (!countryCode || !regex) return undefined;
+  return String.fromCodePoint(
+    ...countryCode
+      .toUpperCase()
```

Comment:
`.toUpperCase()` might not be required now since the regex will catch any codes that aren't uppercase and return undefined.

Would you prefer to leave it in, or remove it?

Reply:
I'm also wondering if the `countryCode` param should be `country`, since that's how it's referred to else where.